### ISSUE TITLE
Callisto Network monetary policy update.

### ISSUE THREAD

**Issue by Dexaran** [EthereumCommonwealth/Roadmap#56]
# Callisto Monetary Policy

The initial [monetary policy](https://en.wikipedia.org/wiki/Monetary_policy) of CLO was temporarily and not suitable for long-term. The main goals of high emission monetary policy were:

- avoid the effect of price collapse right after the airdrop (watch EOS airdrops as an example of what could happen)

- attract miners and initiate Cold Staking

- reduce the influence of the airdropped coins owners to prevent their dominance in voting system

Now, as these goals can be considered achieved, it is necessary to move forwards and design a new monetary policy which will ensure long-term sustainability of CLO. 
I believe that monetary policy is life-important for the project and we must approach this issue with all the responsibility, understanding of fundamental principles and psychological aspects of crypto markets and stock trading.

# Callisto Network fundamentals

>Callisto Network is a project of Ethereum Commonwealth that’s been  researched and designed, and has developed a reference implementation of a self-sustained, self-governed, self-funded blockchain ecosystem.  It also improves the security of the ecosystem of crypto industry by improving smart-contract development methods and environment. (Quote: Callisto Network whitepaper)

>Callisto aims to establish a secure and contribution-friendly environment for further protocol development and improvements. It will rely on a built-in system of smart-contracts to achieve this goal. (Quote: Callisto Network whitepaper )

Callisto Network whitepaper can be found [here](http://whitepaper.callisto.network).

The core features of Callisto are:

- Security Auditing departments (paid from Treasury)

- Development team (paid from Treasury)

- Treasury

Summarizing the above it can be concluded that one of the main goals of Callisto monetary policy is to ensure the viability of the system by maintaining the [liquidity of Treasury funds](https://en.wikipedia.org/wiki/Accounting_liquidity), i.e. ensuring the ability of Treasury to be used for further development of the project.

It should be noted that Treasury is not a storage of value. The main goal of Treasury is to provide a source of funding for Callisto ecosystem, but not to accumulate wealth. Therefore, only Treasury income makes sense when designing a long-term monetary policy, but not the amount of funds which is already stored is Treasury.

# Economical background and market research

## 1. Demand is of decisive importance

The price and market capitalisation of an asset is determined by [supply & demand](https://en.wikipedia.org/wiki/Supply_and_demand). However, when talking about cryptocurrency markets, we should assume that demand is of decisive importance.

We can see plenty of high supply assets with high capitalisation (and high price) and a lot of low supply assets with low price. Some projects are just "big" and the other are just "another low profile coins" with low supply and low price. The destiny of a project is determined by how potential buyers evaluate it, thus the total success of a project is determined by market participants expectations.

In some cases, if you reduce the total supply then you will just reduce the market capitalization of an asset in long term. If there are no additional incentives to buy to ensure the demand (buy pressure) then the supply reduction will not directly affect the price. A good example is Ethereum Classic:

![ETC](https://user-images.githubusercontent.com/26142412/51289154-94d47680-1a18-11e9-8778-5090a1970db0.png)

After the implementation of [ECIP-1017](http://ecip1017.com/) we can observe a short-term price surge (which was also caused by Callisto Airdrop which occurs on [5 March 2018](https://twitter.com/cnbcfastmoney/status/969348370161643521)) followed by a long downtrend. The max total supply of ETC decreased from infinity to 233M, the inflation decreased as well. However, the price has not increased and currently remains lower than at the time of the implementation of the monetary policy.

Lesson is learned: Do not expect price surge when at wrong market phase and do not expect the increase of "demand" when you reduce the "supply".

## 2. Psychology of market cycles

[Market cycles](https://en.wikipedia.org/wiki/Stock_market_cycles) are known phenomenon in stock trading. Read [this](https://fifthperson.com/psychology-market-cycles/) statements for better understanding.

"Wall Street Cheat Sheet" illustration of what market cycles look like:

![image](https://user-images.githubusercontent.com/26142412/51280509-41066500-19f9-11e9-9433-5852b2b417c0.png)

In crypto asset trading, market cycles are also important. All processes are cyclic and iterative. The hardest part of the cycle for investors and projects is the [bearish phase](https://www.investopedia.com/terms/b/bearmarket.asp) ("Anger" and "Depression" on the image above). The effect of fundamental analysis is minimal, everything is stagnating no matter what. The outcome of positive news and successful development is zeroed out by lack of buyers.

![market_cycles](https://user-images.githubusercontent.com/26142412/51291533-e84ac280-1a20-11e9-918c-e78e24f2a40f.png)

According to the chart above, it can be concluded that the length of a single market cycle in crypto industry is about 260 days.

## 3. Buyers

There are multiple kinds of participants in crypto market:

1. Newcomers

2. Professional traders (TA traders)

3. Investors/bagholders

Newcomers enter the crypto market during the mid-late bullish phase (Optimism, belief, thrill, euphoria on the image above). They invest in whatever they can without any research. Professional traders do not care about fundamental analysis because they trade charts. You can join a [CryptoUB's discord group](https://discord.gg/RKJ52EJ), [Crypto cartel group](https://t.co/6ps4uwZZXv) or [Luke Martin's group](https://t.me/joinchat/FfNHd0OzZObfoiOVmyoQdg) and scroll the history of messages to see how traders think.

Only a certain group of buyers evaluate fundamental analysis of an asset before buying it.

## 4. Inflation reduction

Callisto is based on the ["gym theory of markets"](https://docs.google.com/document/d/1LpRHzW7gGV1JfiXUQOmW-UEVF9TCNNWrYiItDuLgiGA/edit). According to this theory, it is important to keep holders incentivized to hold. Cold Staking is intended to incentivize buyers to hold, while high inflation is incompatible with this theory. It is important to keep inflation levels low enough to ensure holding incentivization.

A common argument against monetary inflation in cryptoasset networks like Bitcoin is the ‘sound money’ thesis. Sound money is generally defined as: money that has a purchasing power determined by markets, independent of governments and political parties — such as precious metals like Gold when used as a medium-of-exchange.

ZCash is also a good example of inflation impact.

![zec_inflation](https://user-images.githubusercontent.com/26142412/54084810-ed582e00-434e-11e9-8bbb-5dca5da120a4.png)

ZCash is the first crypto asset which implements [zk-snarks](https://z.cash/technology/zksnarks/), however, it is now perceived only as an opportunity to make an anonymous transaction. In general, high inflation is a strong argument against long term holding, which is against the purpose of Cold Staking of Callisto.

# Designing the monetary policy

Taking the above statements into account, we can submit a set of rules which the desired monetary policy must match:

1. The main goal of the monetary policy is to ensure longterm sustainability of Callisto Network.

2. Inflation rate must be reduced.

3. Previously announced max cap must not be changed. The max cap of CLO is still 6,5 B coins.

4. It is necessary to reduce inflation no more than once per market cycle to ensure that the ecosystem can rebalance.

5. A radical decrease in inflation is unacceptable since it would entail a decrease in hashrate.

6. Significant decrease in inflation during the bearish market cycle phase is even more harmful as it entails the reduction of Treasury income and endangers the viability of Callisto ecosystem and the existence of Callisto Team.

# Proposal

1. Reduction stage (starts at block 2,750,001): reduce block reward by 40% each 1,500,000 blocks (~250 days) starting at 2,750,001 block and do it 4 times. Then reduce the block reward by 35% so that block reward becomes 50,54 CLO in 2022 year. Reduce mining reward percentage by 5% and increase Cold Staking reward percentage by 5% at each block reward reduction.
2. Main stage (starts at block 10,250,001): during the main phase, block reward will remain the same. The existence of main phase is required because the situation in crypto industry is changing quickly. Thus, it will change before the final stage and it may require Callisto Team to react taking into account the upcoming events.
3. Final stage (starts at block 55,250,001): reduce block reward by 2% each 500,000 blocks starting at 55,250,000 block (2040 year). This is necessary to preserve the announced max cap of Callisto.

A full table with detailed numbers can be found here: https://docs.google.com/spreadsheets/d/1WtxDprdbxO0Xmm4dBWeazvnT6fp_xYXw6fe3hCDBpCk/edit#gid=1751251200

| Start Block | End Block | Estimate End Date | Block Reward Reduction, % | Block Reward | Total CLO | Treasury income/month | Mining, % | Cold Staking, % |
| ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: |
| 1 | 2 750 000 | 25.05.2019 | 0 | 600 | 1 750 000 000 | 10 500 000 | 70 | 20 |
| 2 750 001 | 4 250 000 | 28.12.2019 | 40 | 360 | 2 290 000 000 | 7 464 960 | 65 | 25 |
| 4 250 001 | 5 750 000 | 01.08.2020 | 40 | 216 | 2 614 000 000 | 4 478 976 | 60 | 30 |
| 5 750 001 | 7 250 000 | 06.03.2021 | 40 | 129,6 | 2 808 400 000 | 2 687 386 | 55 | 35 |
| 7 250 001 | 8 750 000 | 09.10.2021 | 40 | 77,76 | 2 925 040 000 | 1 612 431 | 50 | 40 |
| 8 750 001 | 10 250 000 | 03.03.2022 |  35 |  50,54 | 3 000 856 000 | 1 048 080 | 45 | 45 |
| 10 250 001 | 55 250 000 | 11.03.2040 |  0 |  50,54 | ... | 1 048 080 | 45 | 45 |
| ... | ... | ... | ... | ... | ... | ... | 45 | 45 |
|55 250 001 | 55 750 000 | 22.05.2040 | 2 | 49,52 | 5 300 102 560 |  | 45 | 45 |
|55 750 001 | 56 250 000 | 2.08.2040 | 2 | 48,54 | 5 324 373 789 |  | 45 | 45 |
| ... | ... | ... | ... | ... | ... | ... | ... | ... |
|173 250 001 | 173 750 000 | 17.02.2087 | 2 | 1 | 6 503 350 273 |  | 45 | 45 |
| ... | ... | ... | ... | ... | ... | ... | ... |
|210 250 001 | 210 750 000 | 15.10.2101 | 2 | 0,09 | 6 511 351 162 |  | 45 | 45 |

# References

1. [Monetary policy](https://en.wikipedia.org/wiki/Monetary_policy)

2. [Microeconomics](https://en.wikipedia.org/wiki/Microeconomics)

3. [Longterm holding strategy](https://www.investopedia.com/terms/b/buyandhold.asp)

4. [The importance of psychology in trading](https://www.investopedia.com/articles/trading/02/110502.asp)

5. [On Cryptoasset Monetary Inflation](https://medium.com/mosaic-network-blog/on-cryptoasset-monetary-inflation-766d973670bf)

6. [ZCash emission reduction](https://unhashed.com/cryptocurrency-news/zcash-plans-reduce-inflation-harmony-mining/)

**Comment by workvint** [comment]
@Dexaran How about reducing the fixed payment per month of *500 000* CLO (for the founders of the project) by the same percentage as the miners? 

Perhaps this would be a good taste.

**Comment by Mixavid** [comment]
With the introduction of a new monetary policy reward for cold stake, after a few years will be very small at the rate of one coin, consider the variation in the allocation of cold stake of a percentage of the income of the project ( for example, from the accelerated verification of the contract and other revenues that will have the project in a few years, with the introduction of new services). Not at the moment, but when the project becomes larger and will generate income comparable to mining.

**Comment by kirilusd** [comment]
This monetary policy is perfect. I prefer to start from 2500000.

**Comment by DmZak** [comment]
Needs more % for investors ( Coldstakers ) to attract them. After Pirl implementation hashrate can be lowered. 3% in month during 2019 will be very attractive. You may claim, that you guarantee 2-3% during a year, without additional calculations from investor side. Too much % for miners, because miners are not straight investors and not interested in coin future

**Comment by workvint** [comment]
@DmZak 
> 3% in month during 2019 will be very attractive.

Indeed it is. 

But from start CS in last november CLO lose about  *40%* from BTC and if for this 4 month your CS got back this dump then you lucky.

And if you really look at the percentage, then *3%* can quickly fall to *2%* per month.

For two weeks, the number of shareholders has increased by 50 million CLO and the percentage you receive if you pick up now is *3.4%*, and two weeks ago it was *3.7%*.

**Comment by DmZak** [comment]
> 
> 
> @DmZak
> 
> > 3% in month during 2019 will be very attractive.
> 
> Indeed it is.
> 
> But from start CS in last november CLO lose about _40%_ from BTC and if for this 4 month your CS got back this dump then you lucky.
> 
> And if you really look at the percentage, then _3%_ can quickly fall to _2%_ per month.
> 
> For two weeks, the number of shareholders has increased by 50 million CLO and the percentage you receive if you pick up now is _3.4%_, and two weeks ago it was _3.7%_.

I bought CLO in June  and you know price )) , but I believe in project and try to do it better? as I imagine perspective)

**Comment by workvint** [comment]
@Dexaran 

Does the team have a plan in case the price does not respond to a decrease in the block reward?

Is a more flexible revision of monetary policy possible in the future, for example, not once a year, but depending on conditions?

**Comment by bokel1** [comment]
@Dexaran Given the current state of the network and the expectations for the future of the project, a payroll of 500k CLO per month is rather controversial. How about stating dev's payroll from the treasury in fiat and not in CLO. Say, 10kUSD for the CEO, 10kUSD for the CRO, and so on. 
500k CLO salary per month brings rather unfair power distribution between devs, miners and investors (holders and stakers).
It is fair to say that devs have started it all, miners made it possible to run, but the community will make it or break it...

**Comment by HomeOfThePurist** [comment]
Well, first, good job CLO Network, you are very welcome in my world. I love the project.

First things first, about the monetary plan, you first need to understand that you have to get out of the "old world". This is the real challenge. You have to understand that if Cryptocurrencies have to succeed, the old system have to be recognized as obsolete.

In innovation class, our teachers told us that whatever the resistance, mankind can't resist a "product" that is 10 times more efficient that the products it is supposed to replace.

Why 10 times? I don't know, who cares, really? We can't always measure it. The thing to keep in mind is that if you are very efficient, you replace the obsolete technology. End of story. And people will be forced to adopt the new technology.

Who here is still paying people to carry water to their homes? No. We all have a plumbing system to get as much water as we want.
Well, banks and the old banking system, the old financial system is the guy carrying the water. 

And look at all the smart people in here explaining the new strategy using old concepts that failed. We even have a wonderful "Wall Street Cheat Sheet" in the thread. Very nice. Very sophisticated. But it is thinking in the box, so let's rejects all of this. 

Let's think outside the box for a minute.

Supply & Demand you say? What about a guaranteed minimum Demand? Take a look at the USDT. It's a stable coin. Meaning, you won't lose money on it... But the problem, is that 1) The value is set on a FIAT currency. If tomorrow the USD comes crashing, the USDT holders will lose money. 2) You won't make money on it. It will always be worth the same amount of USD.

Hear me out. I'm talking about people confident in the value of their CLOs. A minimum value that will follow the trading value upward, but never downward.

We need an entity, not necessarily centralized, to control the Network. The first mission of that Network as an entity would be to set a minimum value. This minimum value will only go up. Never down. In worse case scenarios, it stays stable. 

This is the safety we can give to investors. If you buy your CLOs at the set price by the Network, you won't lose money.  Whatever what you do, you will get out with the value you invested in.

CLOs should be like a rice and corn bags in a cellar. As long as there is no rats eating them, they are safe. Whatever happens, you won't starve to death. 

If you mix that with the Cold Staking benefits, CLO will only be limited by the implemented technology.

People are shocked by the sums lost by some in Cryptocurrencies. We need stability and profits. We need a revolution. 

Second, we will need security. People will make mistakes. Shit happens. Dishonest people will do something. We will need tools to retrieve the lost amounts. 

Once these tools will be available, the criminals will of course exchange the stolen CLOs into other currencies. But as the stolen CLOs will be seized and sent back to the rightful owners, the loss will be on the people buying stolen CLOs from unknown vendors through exchanges. Forcing people to never buy CLOs unless it comes the Network's approved vendors.

Of course, I'm being extremely simplistic here, but I can dive into details if necessary.

So, in short. SECURITY.

Financial security and General security.

**Comment by vinnykahlon** [comment]
Reduction should start at 2500000 and POS or cold staking should be given preference. Less than 3 % reward would not be good enough for investors to keep staking . So it should seriously thought that over 3 % reward is important to keep stakers interested. Reduce rewards for minera increase for stakers.

**Comment by workvint** [comment]
@kirilusd @vinnykahlon Where did block number *2 500 000* come from?

You did the calculations as a team, built graphs, maybe you know another market cycle in crypto industry?

**Comment by MyCryptoOdyssey** [comment]
> @Dexaran Given the current state of the network and the expectations for the future of the project, a payroll of 500k CLO per month is rather controversial. How about stating dev's payroll from the treasury in fiat and not in CLO. Say, 10kUSD for the CEO, 10kUSD for the CRO, and so on.
> 500k CLO salary per month brings rather unfair power distribution between devs, miners and investors (holders and stakers).
> It is fair to say that devs have started it all, miners made it possible to run, but the community will make it or break it...

I tend to agree with this perspective. This is somewhat counter to the idea of cryptocurrency as a replacement to FIAT but it is a more stable representation of compensation. This does create more predictability in salary for the team.  I however think $10K is not worth while for the devs and it would need to be a higher amount.

**Comment by kirilusd** [comment]
I think here many people do not realize that $ CLO exists thanks to the efforts of Dex, Yohan and others from the team. Why do you even comment on his salary. Indeed a 10k usd salary is ridiculous for the position and function of the management team. I suggest that, regardless of the price of CLO, Dex and Yohan's salary remains 500,000 CLO. This is exactly the idea and incentive for these people to develop this coin because they actually make their personal income and incentive. Everything else I guess is going to pay for staff and other expenses.

**Comment by MyCryptoOdyssey** [comment]
> > @Dexaran Given the current state of the network and the expectations for the future of the project, a payroll of 500k CLO per month is rather controversial. How about stating dev's payroll from the treasury in fiat and not in CLO. Say, 10kUSD for the CEO, 10kUSD for the CRO, and so on.
> > 500k CLO salary per month brings rather unfair power distribution between devs, miners and investors (holders and stakers).
> > It is fair to say that devs have started it all, miners made it possible to run, but the community will make it or break it...
> 
> I tend to agree with this perspective. This is somewhat counter to the idea of cryptocurrency as a replacement to FIAT but it is a more stable representation of compensation. This does create more predictability in salary for the team. I however think $10K is not worth while for the devs and it would need to be a higher amount.

Just to be clear I was responding to @bokel1. Actually given the current valuation of Callisto I think 500K Callisto is not enough. My point was to try and have a predictable method for paying the Team. I think we are on the same page there and I understand that using $CLO as the pay type that they would have more skin in the game (but hard to live your life without a predicable paycheck ergo my comment on tying compensation to a FIAT).

**Comment by Frozenhead007** [comment]
A few comments on shorting the miners. What happens when Clo is not worth mining? This is a proof of work coin right? I’m a miner supporting the network I’m currently paying $600 a month in electricity and get just enough Clo to cover that. If you short the miner who will uphold your network? I can always move on to something else but what will happen to Clo? I am for less inflation in the coin but that balance needs some thought. Also the 3% a month is crazy high! 36% a year actully more ! Lol that will never work out. How many real world investments that are legit pay that? You need some real help in economics in this area. Unless you what to pump this thing a couple years and dump it.

**Comment by DmZak** [comment]
> 
> 
> A few comments on shorting the miners. What happens when Clo is not worth mining? This is a proof of work coin right? I’m a miner supporting the network I’m currently paying $600 a month in electricity and get just enough Clo to cover that. If you short the miner who will uphold your network? I can always move on to something else but what will happen to Clo? I am for less inflation in the coin but that balance needs some thought. Also the 3% a month is crazy high! 36% a year actully more ! Lol that will never work out. How many real world investments that are legit pay that? You need some real help in economics in this area. Unless you what to pump this thing a couple years and dump it.

Your $600 is not straight investment in CLO, its your investment in electric company) If hashrate be lowered, dufficulty be lowered too. After Pirl implementation CLO no need crazy high hashrate. I was ASIC miner, but sold everything and bought CLO. Wish CLO become POS coin)))

**Comment by AlexKh40** [comment]
May be make one more wallet to accumulate CLO (get 5% from mining percent, for example)/ This wallet will be activated 03.03.2022 and increase cold stacking treasury every month for fixed count of coins.

**Comment by Frozenhead007** [comment]
Your $600 is not straight investment in CLO, its your investment in electric company) If hashrate be lowered, dufficulty be lowered too. After Pirl implementation CLO no need crazy high hashrate. I was ASIC miner, but sold everything and bought CLO. Wish CLO become POS coin)))



Without a miner creating the coin and upholding the Clo network you would have nothing! This is not a POS coin.  Believe me, having a large community of miners is in Clo favor. Having a small group is not. Look at all the coins who let ASICS take over or just have a small hashrate. You never hear squat about them anymore. I have never sold 1 coin I have mined since day one. Why would I mine a coin that wouldn’t even pay the electric bill to mine it? Those are shit coins and there are thousands of them. Right now I don’t see any huge advances on the blockchain side. What they have done so far is smart to keep coin alive. Getting on many trading platforms and having a staking coin. Making the miners pay to produce the coin and giving it to others who hold the coin. This can be done but a balance of economics needs to be applied or it will fall apart.

**Comment by Frozenhead007** [comment]
Research needs to be done if they really want to make this work in the future for this coin. 3% a month is just insane. It will never work out in the long run. Look at the highest yielding dividend on the DOW. Most of the good ones are 4 or 5% per YEAR. I could see 10 or 15% return for staking (dividend) for this coin and maybe it might not collapse on itself. I have not run their numbers yet because I’m just to busy and I don’t have a couple days to run the numbers and look at their distribution. Hope they know what they are doing or this becomes a pump and dump.

**Comment by DmZak** [comment]
> 
> 
> Research needs to be done if they really want to make this work in the future for this coin. 3% a month is just insane. It will never work out in the long run. Look at the highest yielding dividend on the DOW. Most of the good ones are 4 or 5% per YEAR. I could see 10 or 15% return for staking (dividend) for this coin and maybe it might not collapse on itself. I have not run their numbers yet because I’m just to busy and I don’t have a couple days to run the numbers and look at their distribution. Hope they know what they are doing or this becomes a pump and dump.

Its a crypto market with normal +- 5% exchanging rate a day. We have a lot coins on market with reward from 0.5 to 120% a year. We are not first and biggest in reward. In POS we have a chance to convert all inflation in stakers reward, but in POW our inflation higher, then stakers% 3 times. With reward 4-5% a year, CLO have no interest for investors. In autumn 2019 CLO monthly interest be 2% its a lowest level for investors attraction. In a word: less % for miners, more% for buyers )

**Comment by DmZak** [comment]
> 
> 
> Research needs to be done if they really want to make this work in the future for this coin. 3% a month is just insane. It will never work out in the long run. Look at the highest yielding dividend on the DOW. Most of the good ones are 4 or 5% per YEAR. I could see 10 or 15% return for staking (dividend) for this coin and maybe it might not collapse on itself. I have not run their numbers yet because I’m just to busy and I don’t have a couple days to run the numbers and look at their distribution. Hope they know what they are doing or this becomes a pump and dump.

Its a crypto market with normal +- 5% exchanging rate a day. We have a lot coins on market with reward from 0.5 to 120% a year. We are not first and biggest in reward. In POS we have a chance to convert all inflation in stakers reward, but in POW our inflation higher, then stakers% 3 times. With reward 4-5% a year, CLO have no interest for investors. In autumn 2019 CLO monthly interest be 2% its a lowest level for investors attraction. In a word: less % for miners, more% for buyers )

**Comment by workvint** [comment]
The document attached in the title ([New monetary policy](https://docs.google.com/spreadsheets/d/1WtxDprdbxO0Xmm4dBWeazvnT6fp_xYXw6fe3hCDBpCk/)) describes well what will happen with the CS percentage within 65 years.

For those who are too lazy to read :)

On condition of a freezing of 50% of coins.

* After the first decrease at the end of May of this year, the CS percentage per round will be **2.13 %**.
* At the beginning of 2020 - **1.62 %**.
* At the beginning of 2021 - **0.68 %**.
* ...
---

So coin positioning as a source of passive income is possible for the first three years.

But positioning as a way of maintaining the command is absolutely true.

**Comment by Dexaran** [comment]
@workvint 

>Does the team have a plan in case the price does not respond to a decrease in the block reward?

Well, we are not trying to artificially manipulate the price. We are going to create a long-term sustainable ecosystem and introduce changes that we believe are necessary for further development.

>Is a more flexible revision of monetary policy possible in the future, for example, not once a year, but depending on conditions?

I would say that it is possible but we do not have any plans yet.

**Comment by Dexaran** [comment]
@bokel1 

>@Dexaran Given the current state of the network and the expectations for the future of the project, a payroll of 500k CLO per month is rather controversial. How about stating dev's payroll from the treasury in fiat and not in CLO. Say, 10kUSD for the CEO, 10kUSD for the CRO, and so on.
500k CLO salary per month brings rather unfair power distribution between devs, miners and investors (holders and stakers).
It is fair to say that devs have started it all, miners made it possible to run, but the community will make it or break it...

We do not have any plans to update the payment schedule yet.  
We do not have any plans to update the payments yet. For the moment it will stay the same as it was described in the whitepaper from the very beginning. We will publish updates if this will change, but I don't think that it is related to the monetary policy.

**Comment by Dexaran** [comment]
@HomeOfThePurist 

>First things first, about the monetary plan, you first need to understand that you have to get out of the "old world". This is the real challenge. You have to understand that if Cryptocurrencies have to succeed, the old system have to be recognized as obsolete.

> ...

>And look at all the smart people in here explaining the new strategy using old concepts that failed. We even have a wonderful "Wall Street Cheat Sheet" in the thread. Very nice. Very sophisticated. But it is thinking in the box, so let's rejects all of this.

We should not just discard “old concepts” and invent our own new ones. The global economy does not change quickly and the processes do not occur instantly.

If you think that cryptocurrencies will replace fiat and the banks will fall tomorrow then I will disagree. This is not how things are made in global economy area.

>Second, we will need security. People will make mistakes. Shit happens. Dishonest people will do something. We will need tools to retrieve the lost amounts.

>Once these tools will be available, the criminals will of course exchange the stolen CLOs into other currencies. But as the stolen CLOs will be seized and sent back to the rightful owners, the loss will be on the people buying stolen CLOs from unknown vendors through exchanges. Forcing people to never buy CLOs unless it comes the Network's approved vendors.

One of the main concepts of cryptocurrencies is censorship resistance. Having an entity capable of determining whether the funds are "approved" or not does not correspond to the above concept unless you can guarantee the decentralization of this entity.

**Comment by Dexaran** [comment]
@Frozenhead007 

>A few comments on shorting the miners. What happens when Clo is not worth mining? This is a proof of work coin right?

The coin can not be "not worth mining" if its price is not equal to zero. If the price drops and the hashrate drops in the same ratio then the profitability will remain the same.

In addition we evaluate the switching to POS or POS/POW hybrid consensus. If the hashrate will drop significantly then we will switch the consensus algorithm in the next hardfork.

> I am for less inflation in the coin but that balance needs some thought. Also the 3% a month is crazy high! 36% a year actully more ! Lol that will never work out. How many real world investments that are legit pay that?

The system is intended to be self-balancing. So, the higher the ROI the more stakers will come causing the drop of the ROI.

We are only rebalancing the allocation between miners and stakers.

**Comment by Dexaran** [comment]
@DmZak 

>Its a crypto market with normal +- 5% exchanging rate a day. We have a lot coins on market with reward from 0.5 to 120% a year. We are not first and biggest in reward. In POS we have a chance to convert all inflation in stakers reward, but in POW our inflation higher, then stakers% 3 times. With reward 4-5% a year, CLO have no interest for investors. 

I would say that 4% - 7% ROI is a good target. However, we are not directly determining the ROI with our monetary policy. The actual ROI is determined by the community of stakers and the amount of staked CLOs.

As I've mentioned earlier we only determine the % allocation between miners and stakers.

**Comment by Dexaran** [comment]
The fact that CLO may have that high ROI indicates that we are in the very early stage. Early adopters earn more but their risks are higher.

**Comment by slavserver** [comment]
Dear developers, did your project become of no interest to you? please provide updated information to the pool operators about the monetary policy of your network

**Comment by Dexaran** [comment]
Due to the hard fork on the 2900000 block, we have shifted the reward change dates by 150,000 blocks.

The next reward reduction will happen at block 4,400,000.

**Comment by Spigfish** [comment]
> Dear developers, did your project become of no interest to you? please provide updated information to the pool operators about the monetary policy of your network

Some other awesome news for miners as well!
https://callisto.network/blog/post/callisto-network-introduces-the-dynamic-gas-price-feature/
